modal/_utils/function_utils.py

```diff
@@ -320,16 +320,29 @@ def is_nullary(self):
         return True
 
 
-def method_has_params(f: Callable[..., Any]) -> bool:
-    """Return True if a method (bound or unbound) has parameters other than self.
+def callable_has_non_self_params(f: Callable[..., Any]) -> bool:
+    """Return True if a callable (function, bound method, or unbound method) has parameters other than self.
 
-    Used for deprecation of @exit() parameters.
+    Used for deprecation of @exit() parameters and ensuring @asgi_app / @wsgi_app functions don't have non-default
```

Comment:
Super nit: technically the deprecation on `exit()` has passed but we still want exit functions to be nullary; could take the opportunity to update the docstring here.

Also maybe confusing to mention the "non-default" part of what we use this for vis-a-vis `asgi_app` etc. since we handle the non-default part with a different function.

Reply:
Updated!